backup_and_restore/replacing-unhealthy-etcd-member.adoc

```diff
@@ -0,0 +1,37 @@
+[id="replacing-unhealthy-etcd-member"]
+= Replacing an unhealthy etcd member
+include::modules/common-attributes.adoc[]
+:context: replacing-unhealthy-etcd-member
+
+toc::[]
+
+This document describes the process to replace a single unhealthy etcd member.
+
+This procedure assumes that there is still an etcd quorum in the cluster. The process to replace the unhealthy member depends on whether the etcd member is stopped, or whether it is running but etcd is crashlooping.
```

Comment:
@bergerhoffer I don't understand in what case the member will be stopped, we probably need to reword

```
The process to replace the unhealthy member depends on whether the etcd member is stopped,

Reply:
@alaypatel07 It was meant to be the first procedure for when the node is stopped. Should it be "whether the node is stopped" instead of "whether the etcd member is stopped"? Or is there something else incorrect about this wording?

Reply:
Updated per our slack conversation.